app/lib/presentation/screens/home.dart

```diff
@@ -47,71 +48,58 @@ class Home extends StatelessWidget {
                     padding: const EdgeInsets.all(8.0),
                     child: Row(
                       mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                      children: const [
-                        Text(
+                      children: [
+                        const Text(
                           "Your Accounts",
                           style: TextStyle(
                             fontWeight: FontWeight.bold,
                             fontSize: 20,
                           ),
                         ),
-                        Text('') //placeholder for button
+                        BlocListener<AccountsCubit, AccountsState>(
+                          listener: (context, state) {
+                            if (state is AccountsDeleted) {
+                              // display message
+                              ScaffoldMessenger.of(context).showSnackBar(
+                                SnackBar(
+                                  content: Text(state.msg),
+                                ),
+                              );
+
+                              // refetch user accounts
+                              BlocProvider.of<AccountsCubit>(context)
+                                  .useraccounts();
+                            }
+                          },
+                          child: GestureDetector(
+                            onTap: () {
+                              BlocProvider.of<AccountsCubit>(context)
+                                  .nukeAccounts();
+                            },
```

Comment:
This should show some kind of confirmation dialog to the user. Only process the request when they have admitted they are sure.

Reply:
i have removed the clear all button, working with states in dialogs is painful in flutter apparently, will look for a better way.